<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fun36tj</class>
 <widget class="QDialog" name="fun36tj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {	
	border-image: url(:/rc/res/main_bk.png) 0 00 0 0 repeat;
}
#widget_2{ 
border-image: url(:/rc/res/main_bk.png) 50 00 0 0 repeat;
}
QPushButton {
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 0px;
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.505682 rgba(35, 97, 222, 247), stop:0.539773 rgba(61, 114, 223, 247), stop:1 rgba(93, 142, 239, 238));
	color: rgb(255, 255, 255);
	font: 16px;
}
QGroupBox::title {
  color: rgb(255, 255, 255);
  subcontrol-origin: margin;
  subcontrol-position: top center;
  padding-top: 10px;  
  border-top: 1px solid rgb(255, 255, 255);
  font:bold 14px;
}
QGroupBox {
	border: 1px solid rgb(255, 255, 255); 
    border-radius: 9px;
	padding:2px; font: bold 16px;	
	background-color: rgba(199, 199, 199, 100);
}

QTableWidget::item {
   
}
QHeaderView::section {
    background-color:  rgb(221, 221, 221); 
	font: 18px;
	border: 0px solid #6c6c6c;
	height: 25px;
}

QTableWidget {
border-left: 1px solid rgb(255, 255, 255); 
border-right: 1px solid rgb(255, 255, 255); 
background-color: transparent;
color: rgb(255, 255, 255);
font: bold 14px;
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>51</y>
     <width>801</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btClose">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>结束调机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btItemUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>上一件</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>工单信息</string>
    </property>
    <widget class="QTableWidget" name="tableWidgetInfo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>311</width>
       <height>291</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>106</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>28</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>属性</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数值</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>派工单号</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>总件/当前</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>工单号</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>产品编号</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>产品描述</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>模具编号</string>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>派工数量</string>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>标准周期</string>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>调机次品数</string>
      </property>
     </item>
     <item row="8" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="9" column="0">
      <property name="text">
       <string>调机状态</string>
      </property>
     </item>
     <item row="9" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="btExit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>451</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>调机参数</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>451</width>
       <height>291</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>180</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>81</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar::vertical { 
	 background: transparent;
     width: 81px;
     margin: 30px 0px 20px 0px;
 }
 QScrollBar::handle:vertical {
    background: rgb(85, 170, 255);
    min-height: 50px;
 }

 QScrollBar::add-line:vertical {
     border: 2px solid transparent;     
	 background: rgb(170, 170, 255);
     height: 30px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     border: 2px solid transparent;
     background: rgb(170, 170, 255);
     height: 30px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 2px solid rgb(170, 255, 255);
     width: 9px;
     height: 9px;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid gray;
    font: 14px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    border-style: solid; /* just a single line */
    border-radius: 0px; /* same radius as the QComboBox */
}

QComboBox::down-arrow {
    
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
    border: 2px solid darkgray;
    selection-background-color: lightgray;
}</string>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btItemNext">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>下一件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSave">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btStart">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>开始调机</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
